doc/api/cli.md

```diff
@@ -108,6 +108,9 @@ added: v12.0.0
 Specify the directory where the CPU profiles generated by `--cpu-prof` will
 be placed.
 
+The default value is controlled by the
+[--diagnostic-dir](#--diagnostic-dir=directory) command line option.
```

Comment:
These links are all broken in the generated HTML docs, e.g. this one would be `#cli_diagnostic_dir_directory`

Reply:
ive addressed this PTAL.